drivers/entropy/entropy_gecko_trng.c

```diff
@@ -44,6 +44,8 @@
 #endif	/* _SILICON_LABS_32B_SERIES_2_CONFIG_* */
 #endif	/* CONFIG_CRYPTO_ACC_GECKO_TRNG */
 
+static int entropy_gecko_trng_init(const struct device *dev);
```

Comment:
Did you consider just moving the init function higher up in the c-file to avoid having to do a forward declaration?

Reply:
I considered it and did this instead to minimize the diff. I could move it if that's preferred.